Add console, enable_wait_mode params for the kernel cmdline.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Florin Sarbu <florin@resin.io>

Index: git/include/configs/ts4900.h
===================================================================
--- git.orig/include/configs/ts4900.h
+++ git/include/configs/ts4900.h
@@ -219,7 +219,7 @@
 		"load mmc 0:1 ${loadaddr} /boot/ts4900-fpga.bin; " \
 		"ice40 ${loadaddr} ${filesize}; " \
 		"load mmc 0:1 ${loadaddr} ${uimage}; " \
-		"setenv bootargs root=/dev/mmcblk1p1 rootwait rw ${cmdline_append}; " \
+		"setenv bootargs root=/dev/mmcblk1p1 rootwait rw console=ttymxc0,115200 enable_wait_mode=off; " \
 		"bootm ${loadaddr} - ${fdtaddr}; \0" \
 	"emmcboot=echo Booting from the eMMC ...; " \
 		"bbdetect; " \
